src/cargo/ops/cargo_new.rs

```diff
@@ -992,11 +994,25 @@ fn update_manifest_with_new_member(
             if was_sorted {
                 members.sort_by(|lhs, rhs| lhs.as_str().cmp(&rhs.as_str()));
             }
+            config.shell().status(
+                "Adding",
+                format!(
+                    "`{display_path}` as member of workspace at `{}`",
+                    root_manifest_path.parent().unwrap().display()
+                ),
+            )?
         } else {
             let mut array = Array::new();
             array.push(display_path);
 
             workspace["members"] = toml_edit::value(array);
+            config.shell().status(
+                "Adding",
+                format!(
+                    "`{display_path}` as member of workspace at `{}`",
```

Comment:
This is printing the literal entry being added to the workspace.  I don't think this will be as meaningful (and may even be confusing as they are more likely to see it as a relative path from where they are at) and would suggest the package name.